cmd/repo/repo_set_config.go

```diff
@@ -52,7 +52,7 @@ the config may be a simple string, like a URL, or a json file.
 
 func init() {
 	repoCmd.AddCommand(repoSetConfigCmd)
-	repoSetConfigCmd.Flags().StringP("type", "t", "", "type of repo to add")
+	repoSetConfigCmd.Flags().StringP("type", "t", "", "new type of repo")
```

Comment:
I would add here "One of %v", repos.SupportedTypes" like in repo_add.go

Reply:
done